### DISCUSSION TITLE
Custom model dimensions

### DISCUSSION THREAD

**Discussion by SchyzaTL** [opentrack/opentrack#1613] [Q&A]
Hi, 
Would it be possible to be more specific in the menu Tracker->Model->Custom because Im not sure which is reference point and remaining P2 and P3 in my case . Here is link to my [model](https://imgur.com/a/U26LuzB) of headtrack which Im using.
Thank you for any answer.

**Comment by SchyzaTL** [comment]
Thank you Mr. Halik for your answer but its not the right answer. Im small
producer of Headtrack´s and want to make somethink different as default.

ne 12. 2. 2023 o 10:54 Stanislaw Halik ***@***.***>
napísal(a):

> Your model is a regular clip. You don't need to use the custom model tab.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/opentrack/opentrack/discussions/1613#discussioncomment-4948065>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AV6PRWXOBT27M3XEL7P7GETWXCXNDANCNFSM6AAAAAAUY3UBB4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
S pozdravom

Ing. Tomáš Leško

**Reply by GO63-samara** [reply]
Then it's: https://github.com/opentrack/opentrack/wiki/Manual-calibration-of-PointTracker-1.1-model
